pkg/common/morpc/backend.go

```diff
@@ -391,6 +391,14 @@ func (rb *remoteBackend) writeLoop(ctx context.Context) {
 			}
 
 			if len(futures) > 0 {
+				if retry && !rb.conn.Connected() {
```

Comment:
Reading this loop, why do you need retry at all?

Reply:
Can solve scenarios where the network recovers after a brief outage